### PR TITLE
HSF Naples newsletter

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
First draft, mostly finished, but no mention of a few sessions, viz.,

- Security
- Data Preservation
- Visualisation

### CODE REVIEW

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+<div style="text-align:center; padding:25px"><img src ="/images/naples-ws-liz.jpg" alt="Liz opens the workshop's science programme" /></div>
+
+Eduardo Rodrigues then reviewed the [HSF Community White Paper Roadmap](https://indico.cern.ch/event/658060/contributions/2876976/attachments/1622751/2582923/2018-03-26_WLCG-HSFWorkshopNaples.pdf){:target="_naples_cwp"}
+that was [published last year](https://arxiv.org/abs/1712.06982){:target="_cwp_roadmap_arxiv"}
+and was the major driver for the workshop. He outlined each of the CWP's
+chapters along with its major R&D goals and stressed again the approach
+of working together, re-characterising CWP as *Community Wide Projects*. 
+
+After this great introduction the workshop, and a review of the important
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: introduction TO the ... ?

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+to start, the group will try and ensure that as many basic training needs
+are satisfied in common. Contributions to training should be recognised
+as a part of the career profile of our developers, but long term career
+prospects remain a concern, as with many other areas in HEP.
+
+<div style="text-align:center; padding:25px"><img src ="/images/naples-ws-banquet.jpg" alt="Well fed at the banquet" /></div>
+
+Opening the final session of the workshop was 
+[a talk from Anna Scaife](https://indico.cern.ch/event/658060/contributions/2940455/attachments/1625101/2587580/WLCG-180329.pdf){:target="_naples_ska"}
+from the Square Kilomoeter Array Telescope and AENEAS projects. This is
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: Kilometer

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+one of the other sciences facing a data deluge in the future and there
+can be a very healthy relationship between HEP, which has been recognised
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: Maybe rephrase a bit since one would expect "between A and B" but there's no B. Maybe just between -> with will do.

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+<div style="text-align:center; padding:25px"><img src ="/images/naples-ws-group-photo.jpg" alt="Group Photo" /></div>
+
+In the analysis domain, where workloads are far more diverse than for organised
+production activities, there are many interesting developments to follow-up.
+Systems like SWAN have had success in bringing notebook technology to
+HEP ROOT analysis and now plans R&D to scale up to future challenges. In
+the meantime, data science tools like Apache Spark clusters achieve a
+scale and ease of use which is admirable and our community would like
+to emulate. albeit that this is much harder to achieve with HEP
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: comma before "albeit"

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+plans regular meetings and a follow up at CHEP. Even the tricky question
+of using new languages, outside our comfort zone of C++ and Python, was
+discussed, generating lively discussion.
+
+<div style="text-align:center; padding:25px"><img src ="/images/naples-ws-group-photo.jpg" alt="Group Photo" /></div>
+
+In the analysis domain, where workloads are far more diverse than for organised
+production activities, there are many interesting developments to follow-up.
+Systems like SWAN have had success in bringing notebook technology to
+HEP ROOT analysis and now plans R&D to scale up to future challenges. In
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: I would remove "ROOT" since that's not a requirement.

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+devices easy to program offer the prospect of more wide-spread use
+of this technology in the future. Looking at other reconstruction challenges
+modern hardware could be used more efficiently if the right algorithms
+are found; that might include replacing some traditional programmed
+solutions with *machine learning* that can usually run well on
+wide vector registers, multiple CPU cores or even GPUs. That challenge might also be
+faced more effectively in common through the use of common toolkits,
+such as VecCore, that help code adapt to the changing hardware landscape.
+An intense discussion about the strategy to be used here, and how much specific HEP solutions
+are really required, was had and typifies many of the open questions
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: "was had" -> "was held" or "took place" I reckon.

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+The Software Development session reviewed progress made by the
+HSF's [Packaging Working Group](/activities/packaging.html){:target="_naples_packaging_wg"}. A common
+solution here will be of great benefit to our R&D projects in the future
+as it will allow a wider base of users to more easily test early
+releases. A decision was also made to consolidate information and best
+practice on performance monitoring and tooling, which will help developers
+across the field.
+
+Helping establish best practice for development is one part of the training
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: This paragraph has a few trivial typos and I would put plurals in a couple of places: ... best practices ... training challenges

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+as it will allow a wider base of users to more easily test early
+releases. A decision was also made to consolidate information and best
+practice on performance monitoring and tooling, which will help developers
+across the field.
+
+Helping establish best practice for development is one part of the training 
+challenge our community faces. Ensuring that our software is well written
+and tested is a key part of making code more sustainable for experiments
+that now have multi-decade lifetimes. The training session gathered speakers
+involved in may of the communitiy's training areas, in experiments and
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: in many ... community's

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+from dedicated schools, and discussion about the best way to address the
+needs in the future was had. Agreeing that a solid base was the best place
```
Comment (eduardo-rodrigues) [inline newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: was held.

**Comment by eduardo-rodrigues** [comment]
Hi @graeme-a-stewart, I could only have a quick read as I have very limited time today. I commented on some trivialities inline ... Otherwise the newsletter is looking very good so far.

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md (hunk 7 above)
Comment (graeme-a-stewart) [reply newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: Usually "best practice" is singular.

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - good suggestions. @harveyj1953 also helped with a few points.

**Comment by graeme-a-stewart** [comment]
OK, ready to go now.

File: newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+and measure the effectiveness of our workloads on our grid infrastructures,
+which the two sessions of the *Performance and Cost Modeling* group looked
+at. There will be a lot of follow up to define the real R&D goals as well
+as the way to measure success in such a complex coupled and large-scale
+system. The *Workload Management* teams look to be involved there too
+as the data processing frameworks are a key actor here. Our current systems
+are rather diverse, but there are many shared goals. Using our large 
+distributed computing systems securely, evolving how
+we authenticate and responding to threats was discussed in the *Security*
+session of the workshop.
```
[the remark this answers is not in the document]
Comment (graeme-a-stewart) [reply newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md]: So, I did keep mention of the security session - in the end it would be bad to have only one session that was missing. It's hard to fit into the flow though, so I made it a separate paragraph.
